libs/alcotest_ext/Alcotest_ext.mli

```diff
@@ -0,0 +1,141 @@
+(*
+   Utilities for writing test suites that are compatible with Alcotest.
+
+   Alcotest API:
+   https://mirage.github.io/alcotest/alcotest/Alcotest/index.html
+*)
+
+type output = Stdout | Stderr | Merged_stdout_stderr | Separate_stdout_stderr
+
+(*
+   A test is a name and a function that raises exceptions to signal
+   test failure.
+
+   There are two main recommended ways of writing a test:
+
+   1. With 'assert false':
+
+   Each test may use 'assert false' to indicate that the test doesn't pass.
+   This is the simplest way of failing while also showing the location
+   of the failure. When using 'assert false', you should print the expected
+   value and actual value if applicable.
+
+   2. With 'Alcotest.(check ...)':
+
+   This is a little nicer because the error messages prints something like
+   "Expecting 'foo', got 'bar'". However, this makes tests more complicated
+   to write. If the test already prints the expected value and the actual
+   value as its output, it's just easier to fail with 'assert false'.
+
+   In any case, Alcotest will capture the output (stdout, stderr) of each
+   test and put it in its own file so we can consult it later. Don't
+   hesitate to log a lot during the execution of the test.
+*)
+type 'a t = private {
+  category : string list;
```

Comment:
maybe add in comment an example of category and name, so people understand their use